[Win] Everest docs (#9622)
treeverse/lakeFS: docs/src/reference/mount.md

### Prerequisites

-   lakeFS Cloud account or lakeFS Enterprise Version `1.25.0` or higher.
-   **Supported OS:** macOS (with NFS V3), Linux and Windows (using CFAPI).
-   **Get the Everest Binary:** Everest is a self-contained binary with no installation required. Please [contact us](http://info.lakefs.io/thanks-lakefs-mounts) to get access.

!!! note "Windows Support"

>     Everest supports Windows starting at version <version>.

Reviewer: ```suggestion
    Everest supports Windows starting at version `0.6.0`.
```

---

> ## Everest for Windows (Cloud Filter API)

Reviewer: We need some section about behavior such as cache management: 
- Files are first stored as stubs "placeholders" not having actual content only metadata
- When files are accessed they get "hydrated" their content is stored on device, Subsequent reads serving data directly from local cache without reaching to lakeFS api.
- Currently, any access to the file content (i.e read first byte) will force the full download of the file.
- Eviction policy: The OS will dehydrate (clear content) under storage pressure and After un-mount all the data get's deleted from local device. 
- Current limitation Windows File Explorer UI: might work very slow in many setups it performs recursive indexing of the whole directory.

> Starting at version <version>, Everest mount is available for Windows OS using *Cloud Filter API* (CFAPI)

Reviewer: ```suggestion
Starting at version `0.6.0`, Everest mount is available for Windows OS using *Cloud Filter API* (CFAPI)
```

Currently, Everest for Windows supports read-only operation.

### Requierments

> Everest Mount supports the windows's native Cloud Filter API. No need in additional installations. 

Reviewer: Please add [link](https://learn.microsoft.com/en-us/windows/win32/cfapi/cloud-files-api-portal) to CFAPI docs. 
Please mentioned minimal version: `Starting in Windows 10, version 1709`

> Make sure your Everest version is > <Version> and lakeFS > <version>.

Reviewer: lakefs version irrelevant

> #### Disable Firewall

Reviewer: ```suggestion
#### Skip Scan in Microsoft Protection Preferences (Windows Defender)
```

> To enable Everest run, we must disable firewall for the mounted path.

Reviewer: ```suggestion
AV will try to fully scan all files contents,  please make sure to must disable firewall for the mounted path.
```

everest mount-server <remote_mount_uri> [flags]
```

!!! note "Windows Support"
    Windows users must utilze the `--root` flag

> 

Reviewer: no need in that. 
`mount-server` is an advanced command anyway. 
If someone want's to use it there are many gotchas we don't mention. 
Just keep it simple with the explanation you already added below for the flag `--root`
